package/docker/openmrs/templates/appointment.properties.template

```diff
@@ -0,0 +1,6 @@
+openmrs.auth.uri=http://${OPENMRS_HOST}:${OPENMRS_PORT}/openmrs/ws/rest/v1/session
+openmrs.root.uri=http://${OPENMRS_HOST}:${OPENMRS_PORT}/openmrs/ws/rest/v1
+openmrs.user= ${OPENMRS_ATOMFEED_USER}
```

Comment:
Atomfeed user in appointment properties? Seems unexpected.

Reply:
Yah Gurpreet will be changing it to OPENMRS_SYSTEM_SMS_USER